src/view/ClusterDetailsPanel.ts

```diff
@@ -176,10 +178,16 @@ export class ClusterDetailsPanel {
 						// get managed cluster addons
 						managedClusterAddons = await this._kubeDataLoader.getResources(selectedCluster, "ManagedClusterAddOn");
 						this._panel.webview.postMessage({"managedClusterAddons":JSON.stringify(managedClusterAddons)});
+						// get managed cluster
+						clusterManager = await this._kubeDataLoader.getResources(selectedCluster, "ClusterManager");
+						this._panel.webview.postMessage({"clusterManager":JSON.stringify(clusterManager)});
 					} else {
```

Comment:
theoretically,  the same cluster can be a hub cluster and also managed cluster (by another hub) 
we should check out if klusterlet exists. 
at line 162.
then replace else with
`if(klusterlet.length > 0) { 
   show managed cluster resources  }
`

Reply:
fixed, Thank you